fred2/ignoreordersdlg.cpp

```diff
@@ -101,53 +87,24 @@ BOOL ignore_orders_dlg::OnInitDialog()
 
 	}
 
-	// set the checkboxes for the orders accepted
-	m_num_checks_active = 0;
-	for (size_t order_id : default_orders){
-		// Not enough space to display checkboxes for all comm orders!
-		// Need to add more checkboxes.
-		if (m_num_checks_active >= MAX_CHECKBOXES)
-		{
-			Warning(LOCATION, "Tried to add more orders than FRED2 has checkboxes for them (can be max %d)!", MAX_CHECKBOXES);
-			break;
-		}
-
-		check_boxes[m_num_checks_active].button->SetWindowText(Player_orders[order_id].localized_name.c_str());
-		check_boxes[m_num_checks_active].id = (int)order_id;
-		m_num_checks_active++;
+	CCheckListBox* list = ((CCheckListBox*)GetDlgItem(IDC_IGNORE_ORDERS_LIST));
```

Comment:
Optional: A more descriptive name than `list`, maybe `ignore_orders_list`, would be nice, but it's not super important.